src/pipecat/services/background_noise.py

```diff
@@ -0,0 +1,120 @@
+import asyncio
```

Comment:
can we move the file to `processors/audio` please? Also, instead of `background_noise.py` it could be called `background_audio.py` since noise really has a negative connotation.